app/controllers/api/openid_connect/authorizations_controller.rb

```diff
@@ -0,0 +1,239 @@
+module Api
+  module OpenidConnect
+    class AuthorizationsController < ApplicationController
+      rescue_from Rack::OAuth2::Server::Authorize::BadRequest do |e|
+        logger.info e.backtrace[0, 10].join("\n")
+        error, description = e.message.split(" :: ")
```

Comment:
Useless assignment to variable - `description`. Use `_` or `_description` as a variable name to indicate that it won't be used.